File: wrappers/java/src/test/java/org/hyperledger/indy/sdk/pool/RefreshPoolTest.java
package org.hyperledger.indy.sdk.pool;

import org.hyperledger.indy.sdk.IndyIntegrationTest;
import org.hyperledger.indy.sdk.utils.PoolUtils;
import org.junit.Test;

import static org.junit.Assert.assertNotNull;

public class RefreshPoolTest extends IndyIntegrationTest {

	@Test
	public void testRefreshPoolWorks() throws Exception {
		Pool pool = PoolUtils.createAndOpenPoolLedger();
		assertNotNull(pool);
		openedPools.add(pool);

		pool.refreshPoolLedger().get();
	}
}

File: wrappers/dotnet/indy-sdk-dotnet-test/Wrapper/PoolTests/RefreshPoolTest.cs
﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
using System.Threading.Tasks;

namespace Indy.Sdk.Dotnet.Test.Wrapper.PoolTests
{
    [TestClass]
    public class RefreshPoolTest : IndyIntegrationTestBase
    {
        [TestMethod]
        public async Task TestRefreshPoolWorks()
        {
            var pool = await PoolUtils.CreateAndOpenPoolLedgerAsync();

            Assert.IsNotNull(pool);
            _openedPools.Add(pool);

            await pool.RefreshAsync();
        }       
    }
}
